subcommands/common_config.go

```diff
@@ -176,16 +178,20 @@ func (o RotateCertOptions) AsConfig() client.ConfigCreateRequest {
 }
 
 type SetUpdatesConfigOptions struct {
-	UpdateTag  string
-	UpdateApps string
-	IsDryRun   bool
-	IsForced   bool
-	Device     *client.Device
-	ListFunc   func() (*client.DeviceConfigList, error)
-	SetFunc    func(client.ConfigCreateRequest, bool) error
+	UpdateTag       string
+	UpdateTagGiven  bool
+	InheritTag      bool
+	UpdateApps      string
+	UpdateAppsGiven bool
+	InheritApps     bool
```

Comment:
I'm drawn in all these flags. Although at a UI level it makes sense to have separate flags for this (for the user convenience), I think that for `SetUpdatesConfigOptions` flags we can do it more neat.

For example, an easy way to eliminate a necessity for `UpdateAppsGiven` flag is to use a `*string` type for `UpdateApps`. The code would be roughly the same, but the amount of code friction is smaller.